spec/features/user_upvotes_on_review_spec.rb

```diff
@@ -7,37 +7,34 @@
 } do
 
   describe 'User upvotes on a review' do
-    scenario 'user clicks upvote' do
+    scenario 'user clicks upvote', js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.